README.md

```diff
@@ -974,6 +976,53 @@ List of supported New Relic resources:
     * `newrelic_synthetics_monitor`
     * `newrelic_synthetics_alert_condition`
 
+### Use with Keycloak
+
+Example:
+
+```
+ export KEYCLOAK_URL=https://foo.bar.localdomain
+ export KEYCLOAK_CLIENT_ID=[KEYCLOAK_CLIENT_ID]
+ export KEYCLOAK_CLIENT_SECRET=[KEYCLOAK_CLIENT_SECRET]
+
+ terraformer import keycloak --resources=realms,openid_clients
+ terraformer import keycloak --resources=realms --filter=rabbitmq_realm=name1:name2:name3
```

Comment:
rabbitmq_realm ?

Reply:
copy/paste from the rabbitmq paragraph in README...